app/assets/javascripts/editor/editor.js.erb

```diff
@@ -1083,6 +1092,10 @@ var CodeOceanEditor = {
         this.initializeDeadlines();
         CodeOceanEditorTips.initializeEventHandlers();
 
+        if (this.getStoredViewedPPInfo() == true) {
```

Comment:
We should also do a comparison with three `===` signs. I assume, you chose this comparison because the return value will be `'true'` (as string) rather than `true` (as boolean). That's fine, just keep the string representation.